tools/test-psoxy.sh

```diff
@@ -45,7 +45,7 @@ then
   log "Calling proxy..."
   log "Request: $TEST_URL"
   log "Waiting Response:"
-  awscurl "${curlparams[@]}" --service execute-api --access_key $CALLER_ACCESS_KEY_ID --secret_key $CALLER_SECRET_ACCESS_KEY --security_token $CALLER_SESSION_TOKEN $TEST_URL
```

Comment:
👍 ; ok, yeah, I see that the auth signature is coupled to the service, not just the URL.  That's very strange to me (why should caller be aware of the AWS service used the implement a REST API??), but it's AWS fault not ours.

Reply:
direct calls to the lambdas is the KISS solution that I think will work for 90% of customers, but customer call from Friday does get me worried that API gateways will be appealing to some enterprise customers precisely BC of the complexity and the ability to put a bunch of additional rules/restrictions on the mappings.